spec/teacher_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../teacher'
+
+describe Teacher do
+  before :each do
+    @teacher = Teacher.new('English', 'Clinton Younge', 18)
+  end
+
+  it 'validate the teacher specialization' do
+    expect(@teacher.specialization).to eq('English')
+  end
+
+  it 'validate the teacher name' do
+    expect(@teacher.name).to eq('Clinton Younge')
+  end
+
+  it 'validate the teacher age' do
+    expect(@teacher.age).to eq(18)
+  end
+
+  it 'validate empty list of rentals' do
+    expect(@teacher.rentals).to eq([])
+  end
+end
```

Comment:
As mentioned in the requirements, you should add unit tests for the methods that exist in the `Teacher` class: 
- can_use_services?